sds_data_manager/lambda_code/SDSCode/batch_starter.py

```diff
@@ -347,7 +345,7 @@ def send_lambda_put_event(command_parameters):
         "Source": "imap.lambda",
         "DetailType": "Job Started",
         "Detail": {
-            "file_to_create": "<s3_uri>",
+            "file_to_create_path": "<file_path>",
```

Comment:
I would put the path right after the file throughout here.
```suggestion
            "file_path_to_create": "<file_path>",
```

Reply:
I was trying to match what we called in the database. Should I rename all to that?

Reply:
My main comment is just that we're changing `file` -> `file_path`, not appending `path` to the variables. I think it sounds better that way at least.

So, IMO, yes I think all of the places that were updated to `file_to_create_path` should be renamed to `file_path_to_create`.

Reply:
changed that now.